Makefile.am

```diff
@@ -322,7 +322,7 @@ if SERVICE_AC
 	fi
 endif
 
-dist_man_MANS = qat_init.sh.8 qatmgr.8
+dist_man_MANS = qat_init.sh.8 qatmgr.8 cpa_sample_code.7
```

Comment:
I haven't checked the creation of the Fedora RPM, but I think this will require a small change in the spec file to include this file (otherwise the RPM creation will be broken).

https://github.com/intel/qatlib/blob/240991f2655fb2f889715003b22dfbba89e9b90f/qatlib.spec.in#L101

```
%files         tests
...
%{_mandir}/man7/cpa_sample_code.7*
```

Reply:
Hi @ColinIanKing, just confirming that this change is breaking the build of the rpm on Fedora. This can be tested with `make rpm`.

Can you just add the following change to this PR:
```
diff --git a/qatlib.spec.in b/qatlib.spec.in
index c5c3e9f29f7c..531f14ccda35 100644
--- a/qatlib.spec.in
+++ b/qatlib.spec.in
@@ -123,6 +123,7 @@ exit 0
 %{_datadir}/qat/calgary
 %{_datadir}/qat/calgary32
 %{_datadir}/qat/canterbury
+%{_mandir}/man7/cpa_sample_code.7*

 %files         service
 %{_sbindir}/qatmgr
```

Don't worry about updating the changelog or bumping up the build number on the `Release` filed.

Thanks.